tests/scripts/thread-cert/node_cli.py

```diff
@@ -152,6 +154,54 @@ def destroy(self):
             self.pexpect.expect(pexpect.EOF)
             self.pexpect = None
 
+    def read_cert_messages_in_commissioning_log(self, timeout=-1):
```

Comment:
The logic about extracting message from log by regular expression depends on node_cli's `pexpect` and it works as other functionalities but just more complex. Thus, it needs to and should be a member function of `node_cli`.

It would be helpful to write some small function to separate the message construction from message matching.